app_scraping/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
Add your new model:

```
from django.contrib import admin
from .models import SkiResort

admin.site.register(SkiResort)
```

See https://docs.djangoproject.com/en/2.2/ref/contrib/admin/ for more info.